core/hotspot/cache/lru.go

```diff
@@ -31,7 +31,7 @@ func NewLRU(size int, onEvict EvictCallback) (*LRU, error) {
 	c := &LRU{
 		size:      size,
 		evictList: list.New(),
-		items:     make(map[interface{}]*list.Element),
+		items:     make(map[interface{}]*list.Element, size),
```

Comment:
In default, the size is a so much large number.
Maybe use size doesn't make sense

Reply:
@louyuting Do you mean that it usually don't reach the size?  So what's the recommended hint size?

Reply:
I have changed to 64.